scripts/android/download.sh

```diff
@@ -21,13 +21,13 @@ for TARGET in arm64-v8a armeabi-v7a x86_64
 do
   ARCH_PATH=$TARGET
   if [ $(git tag -l "${OS}_${TARGET}_${TAG_COMMIT}") ]; then
-      git checkout "${OS}_${TARGET}_${TAG_COMMIT}"
+      git checkout "${OS}_${TARGET}_${TAG_COMMIT}" || git checkout $OS/$TARGET
```

Comment:
This should checkout OS/TARGET and pull the latest if there's no binary for our exact TAG_COMMIT.  If this isn't the desired behavior, this should be changed

Reply:
... no it won't, beccause of line 23